kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/kafkaproxyingresses.kroxylicious.io-v1.yml

```diff
@@ -74,6 +76,27 @@ spec:
                       description: "Protocol specifies the network protocol this ingress expects to receive."
                       type: string
                       enum: ["TCP", "TLS"]
+                loadBalancer:
+                  type: object
+                  description: |-
+                    clusterIP specifies that this ingress is for access within the same Kubernetes cluster using 
+                    ClusterIP Kubernetes Services.
+                  required: [ "bootstrapAddress", "advertisedBrokerAddressPattern"]
+                  x-kubernetes-validations:
```

Comment:
maybe we should try and enforce that te addresses don't end with a port specifier?

Reply:
done, I've implemented them as CEL rules as it was simpler to define the illegal regexes than try and craft the positive pattern.